server/command.go

```diff
@@ -231,6 +232,16 @@ func (c *conn) handleRegionHeartbeat(req *pdpb.Request) (*pdpb.Response, error)
 	}, nil
 }
 
+func checkStore(cluster *RaftCluster, storeID uint64) *pdpb.Response {
+	store, _, err := cluster.GetStore(storeID)
+	if err == nil && store != nil {
```

Comment:
Bacause this function only check whether the store exists and is in tombstone state, it does nothing if it can't get the store, I will add a comment for it.